src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1004,14 +996,7 @@ def freeze_encoder(self):
         self.encoder._freeze_parameters()
 
     @add_start_docstrings_to_model_forward(WHISPER_INPUTS_DOCSTRING)
-    @add_code_sample_docstrings(
-        processor_class=_PROCESSOR_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
-        output_type=Seq2SeqModelOutput,
-        config_class=_CONFIG_FOR_DOC,
-        expected_output=_EXPECTED_OUTPUT_SHAPE,
-        modality="audio",
-    )
+    @replace_return_docstrings(output_type=Seq2SeqModelOutput)
```

Comment:
also need `, config_class=_CONFIG_FOR_DOC)`, just like in other modeling files :-) I am a good guy now

Reply:
Seems to work without it, but will add it eitherway

Reply:
it's probably will miss in the rendered doc page (if we don't include). Never try myself, can't say 100% sure.